go.mod

```diff
@@ -51,6 +50,7 @@ require (
 	github.com/mattn/go-isatty v0.0.19 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
 	github.com/mmcloughlin/addchain v0.4.0 // indirect
+	github.com/nspcc-dev/go-ordered-json v0.0.0-20240110123536-1b6ec8f68c97 // indirect
```

Comment:
Why `indirect`? Something is wrong with version update. Use `go get github.com/nspcc-dev/go-ordered-json@45031a9697afa83fa22f59ddfd1030d49ea0207e` for proper update and after this use `go mod tidy`.